src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java

```diff
@@ -81,4 +89,29 @@ public Date deserialize(JsonElement jsonElement, Type type,
             }
         }
     }
+
+    private class StatsDeserializer implements JsonDeserializer<Stats> {
+
+        @Override
+        public Stats deserialize(JsonElement jsonElement, Type type,
+                JsonDeserializationContext jsonDeserializationContext)
+                throws JsonParseException {
+            try {
+                JsonObject stats = jsonElement.getAsJsonObject();
+                Applications app = gson.fromJson(stats.get("CherryPy Applications"), Applications.class);
+                HttpServer server = null;
+                for(Map.Entry<String, JsonElement> entry : stats.entrySet()){
+                    String key = entry.getKey();
+                    if(key.startsWith("CherryPy HTTPServer ")){
```

Comment:
I also personally tend to prefer inlining and refactoring out as constant as soon as I add a second use, but no strong opinion on this one. I would accept either way. @renner?